addons/mail/static/src/components/discuss_sidebar_category/discuss_sidebar_category.xml

```diff
@@ -29,7 +30,7 @@
                         <t t-if="category.addingItemAutocompleteInputView">
                             <div class="o_DiscussSidebarCategory_addingItem d-flex mb-2">
                                 <AutocompleteInput
-                                    className="'o_DiscussSidebarCategory_addingItemInput flex-grow-1'"
+                                    className="'o_DiscussSidebarCategory_addingItemInput form-control flex-grow-1 w-auto mx-4 rounded'"
```

Comment:
Is the `w-auto` really necessary?

Reply:
Actually, neither `flex-grow-1` is needed too 👍